cli/core/validator.go

```diff
@@ -50,5 +120,5 @@ func GenerateValidatorProof(ctx context.Context, eigenpodAddress string, eth *et
 	validatorProofs, err := proofs.ProveValidatorContainers(header.Header.Message, beaconState, validatorIndices)
 	PanicOnError("failed to prove validators.", err)
 
-	return validatorProofs, validatorIndices
+	return validatorProofs
```

Comment:
the `validatorProofs` already contains the validatorIndices. no need to return 2 items here